### DISCUSSION TITLE
Lesson 4: error message when using chain_id in buildTransaction()

### DISCUSSION THREAD

**Discussion by nargeslein** [smartcontractkit/full-blockchain-solidity-course-py#461] [Q&A] [answered]
Hello everyone,
I am getting the below error message which is fixed only when the parameter chain_id in buildTransaction() is removed.
![image](https://user-images.githubusercontent.com/33735545/143948567-aaa95ee3-2011-4f7f-bbeb-f77fc5b1ed9d.png)

the fix is:
![image](https://user-images.githubusercontent.com/33735545/143948967-5ab24ef2-b53e-4ab1-b4cf-fb97495df923.png)

Any reasons for this?
Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @nargeslein it is because you have a typo, you wrote:

`"chain_id": chain_id` and the correct form is:
` "chainId": chain_id,`.

Give it a try, best regards, Cromewar

**Reply by nargeslein** [reply]
Bugger...many thanks!

**Reply by Daruma1800** [reply]
lol thanks for this, was on this for like 30mins
